price_service/server-rust/src/config.rs

```diff
@@ -0,0 +1,55 @@
+use {
+    libp2p::Multiaddr,
+    std::{
+        net::SocketAddr,
+        path::PathBuf,
+    },
+    structopt::StructOpt,
+};
+
+/// StructOpt definitions that provides the following arguments and commands:
+///
+/// Some of these arguments are not currently used, but are included for future use to guide the
+/// structure of the application.
+#[derive(StructOpt, Debug)]
+#[structopt(name = "pythnet", about = "PythNet")]
+pub enum Options {
+    /// Run the PythNet P2P service.
```

Comment:
price service ?

Reply:
I actually think we should move away from thinking of this software as the price service (though adding it as price_service doesn't help that effort, I will outline why in a README I'll add).